package-lock.json

```diff
@@ -1,8 +1,762 @@
 {
     "name": "djinni-syntax-highlighting",
     "version": "0.0.0",
-    "lockfileVersion": 1,
+    "lockfileVersion": 2,
```

Comment:
The changes in the package-lock.json look pretty wild because I have updated npm on my machine. The new format is backwards compatible, though.